easybuild/easyconfigs/c/CD-HIT/CD-HIT-4.6.6-foss-2016b.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+
+easyblock = 'MakeCp'
+
+name = "CD-HIT"
+version = "4.6.6"
+
+homepage = 'http://weizhong-lab.ucsd.edu/cd-hit/'
+description = """ CD-HIT is a very widely used program for clustering and 
+ comparing protein or nucleotide sequences."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['https://github.com/weizhongli/cdhit/archive/']
+sources = ['V%(version)s.tar.gz']
```

Comment:
@SethosII Are you sure using this tarball is OK, since `cd-hit-v4.6.6-2016-0711.tar.gz` is available as well...

Have you checked that both tarballs are identical? It's possible `cd-hit-v4.6.6-2016-0711.tar.gz` is not a clean copy of the copy, but preprocessed somehow...